src/components/dialogs/network-modifications/voltage-level/creation/voltage-level-creation-dialog.jsx

```diff
@@ -191,18 +191,26 @@ const VoltageLevelCreationDialog = ({
     isUpdate,
     editDataFetchStatus,
     onCreateVoltageLevel = createVoltageLevel,
+    isAttachementPointModification = false,
+    overrideTitle = null,
     ...dialogProps
 }) => {
     const currentNodeUuid = currentNode?.id;
     const { snackError, snackWarning } = useSnackMessage();
-
+    const [isAttachementPointModificationState] = useState(isAttachementPointModification);
     const formMethods = useForm({
         defaultValues: emptyFormData,
         resolver: yupResolver(formSchema),
     });
 
     const { reset, setValue, getValues, watch, trigger, subscribe } = formMethods;
 
+    useEffect(() => {
+        if (isAttachementPointModificationState) {
+            setValue(ADD_SUBSTATION_CREATION, true);
+            setValue(NOMINAL_V, 0); // need a value to pass yup but not taken into account for attachmentLine
```

Comment:
I dont think it is a good idea to init NOMINAL_V as 0 to by pass yup, should customize yup resolver